certified-connectors/Narvar/readme.md

```diff
@@ -0,0 +1,20 @@
+# Title
+Narvar connector with Dynamics 365 Inteligent Order Management provides possibility to provide delivery information to end users.
+
+## Prerequisites
+First, contact your Narvar Customer Support Manager (or email: partnerships@narvar.com) to enable Microsoft Dynamics 365 Intelligent Order Management for your account.
+
+## Supported Operations
+The connector supports the following operations:
+* ```Post-CreateUpdate_Orders```: This will create a new order or send new information about an existing order
+* ```Get-Retrieve_Order```: This operation is used to retrieve order details
+* ```Put-CreateUpdate_Shipments```: This will update an order with a shipment record or change
+
+## Obtaining Credentials
+Once enabled, your credentials will be available within Narvar Hub for you to complete the app installation. ​
+
+## Getting Started
+Configure the connection with the "Account ID" and "Auth Tokens" from Narvar Hub
+
+## Known Issues and Limitations
+This connector was specifically developed to be used for Dynamics 365 Intelligent Order Management's Narvar's provider integration. Support for usecases outside of Dynamics 365 Intelligent Order Management will be limited.
```

Comment:
@BekaAre all files should be free of grammatical errors and typos. here you have "...usecases...", maybe you meant to say "...use cases..."?